src/Styleguide/Components/AuctionResultItem.tsx

```diff
@@ -77,7 +71,9 @@ export const ExtraSmallAuctionResultItem = (props: AuctionResultItemProps) => {
           </Box>
 
           <Box pt={1}>
-            <Serif size="2">Sale: {props.salePrice}</Serif>
+            <Serif size="2">
+              {props.salePrice && `Sale: ${props.salePrice}`}
+            </Serif>
```

Comment:
This can be hoisted up a bit so that an empty dom node doesn't render:

```jsx
{props.salePrice &&
  <Serif size="2">
     ...
  </Serif>
}
```

Reply:
Good call.